gsuite_reports_rules/gsuite_drive_visibility_change.py

```diff
@@ -1,42 +1,123 @@
 from panther_base_helpers import deep_get
 from panther_base_helpers import gsuite_parameter_lookup as param_lookup
 
+EXCLUDED_DOMAINS = ["runpanther.io"]
+
+VISIBILITY = ["people_with_link", "public_on_the_web", "shared_externally", "unknown"]
+
+DOC_TITLE = "<UNKNOWN_TITLE>"
+TARGET_USER_EMAIL = "<UNKNOWN_USER>"
+TARGET_DOMAIN = "<UNKNOWN_DOMAIN>"
+ACCESS_SCOPE = "<UNKNOWN_ACCESS_SCOPE>"
+NEW_VISIBILITY = "<UNKNOWN_VISIBILITY>"
```

Comment:
In order to use the global dictionary (and therefore avoid the use of globals), you could do something like this (also as a nitpick: let's alphabetize it):

```suggestion
ALERT_DETAILS = {
  "ACCESS_SCOPE": "<UNKNOWN_ACCESS_SCOPE>",
  "DOC_TITLE": "<UNKNOWN_TITLE>",
  "NEW_VISIBILITY": "<UNKNOWN_VISIBILITY>",
  "TARGET_USER_EMAIL": "<UNKNOWN_USER>",
  "TARGET_DOMAIN": "<UNKNOWN_DOMAIN>",
  }
```

Then down below you can reference this dictionary by keys without using `global`.